pytest.ini

```diff
@@ -1,3 +1,4 @@
 [pytest]
 markers =
     slowtest
+    build
```

Comment:
* I don't really like the name of this marker. It doesn't really explain what the marker stands for. 
* Next to that I don't really see the added value of a marker here. As far as I can see the only goal of this marker is to make sure that the `test_build_docs` test runs on its own. Why not run it explicitly via `pytest docs/check_sphinx.py::test_build_docs`? Or am I missing something?

Reply:
I was mostly thinking that if other doc tests are added, it's easier to seperate the linkcheck.
Doing the inverse, where we markt eh linkcheck is probably better, and having tox run "not linkcheck".
The linkcheck mark would also be clearer in what it is marking?

Reply:
Good idea. Let's implement it like that.

Reply:
implemented. Tested here: https://jenkins.inmanta.com/job/core/job/inmanta-core-test-docs/422/